agents-docs/content/docs/talk-to-your-agents/api.mdx

```diff
@@ -71,12 +71,16 @@ The response is an SSE stream of JSON events compatible with the Vercel AI SDK U
 
 ### Event Types
 
+#### Text Streaming Events
 - **`text-start`** — Indicates a new text segment is starting
-- **`text-delta`** — Carries the text content delta for the current segment
+- **`text-delta`** — Carries the text content delta for the current segment  
 - **`text-end`** — Marks the end of the current text segment
+
+#### Data Events
 - **`data-component`** — Structured UI data emitted by the agent (for rich UIs)
-- **`data-artifact`** — Artifact data emitted by tools/agents
-- **`data-operation`** — Operational events (status updates, completion, errors)
+- **`data-artifact`** — Artifact data emitted by tools/agents (documents, files, saved results)
+- **`data-operation`** — Low-level operational events (agent lifecycle, completion, errors)
+- **`data-summary`** — AI-generated status updates with user-friendly labels and contextual details
 - **`error`** — Error message emitted by the server/agent
```

Comment:
@tim-inkeep can you look into the difference between `error` and `data-operation` errors? I'm curious if there are now duplicate error reporting objects documented.